react/src/pages/analysis/Analysis.tsx

```diff
@@ -152,48 +198,92 @@ export default function Analysis() {
                 <MaterialTable
                     columns={[
-                        { title: 'AnalysisID', field: 'analysisID', type: 'string' },
-                        { title: 'Submitted', field: 'dateSubmitted', type: 'string' },
-                        { title: 'Submitted By', field: 'submittedBy', type: 'string' },
-                        { title: 'Project', field: 'project', type: 'string' },
-                        { title: 'Participants', field: 'participants', type: 'string', render: rowData => rowData.participants.join(', ') },
-                        { title: 'Pipeline', field: 'pipeline', type: 'string' },
-                        { title: 'Status', field: 'status', type: 'string' }
+                        { title: 'Analysis ID', field: 'analysis_id', type: 'string', editable: 'never', width: '8%' },
+                        { title: 'Pipeline ID', field: 'pipeline_id', type: 'string', editable: 'never', width: '8%' },
+                        { title: 'Assignee ID', field: 'assignee', type: 'string', editable: 'never', width: '8%' },
+                        { title: 'Requester ID', field: 'requester', type: 'string', editable: 'never', width: '8%' },
+                        { title: 'Updated', field: 'updated', type: 'string', editable: 'never' },
+                        { title: 'Result HPF Path', field: 'result_hpf_path', type: 'string' },
+                        { title: 'Status', field: 'state', type: 'string', editable: 'never', defaultFilter: chipFilter },
+                        { title: 'Notes', field: 'notes', type: 'string', width: '30%', /* render: renderNotes, */
+                        editComponent: props => (
+                            <TextField
+                            multiline
+                            value={props.value}
+                            onChange={event => props.onChange(event.target.value)}
+                            rows={4}
+                            fullWidth
+                            />
+                        )}
                     ]}
-                    data={analyses}
+                    data={rows}
                     title={
                         <Title>Active Analyses</Title>
                     }
                     options={{
                         pageSize: 10,
                         filtering: true,
-                        search: false
+                        search: false,
+                        padding: 'dense'
                     }}
                     actions={[
                         {
-                            icon: VisibilityIcon,
+                            icon: Visibility,
                             tooltip: 'Analysis details',
                             onClick: (event, rowData) => {
-                                setActiveRow((rowData as AnalysisRun))
+                                setActiveRow((rowData as AnalysisRow))
                                 setDetail(true)
```

Comment:
For consistency with how the notification panel changes the address bar, this should also be adjusted to use the history API.